app/models/prompt.rb

```diff
@@ -29,7 +29,7 @@ class Prompt < ApplicationRecord
   accepts_nested_attributes_for :optional_tag_set
   has_many :optional_tags, through: :optional_tag_set, source: :tag
 
-  has_many :request_claims, class_name: "ChallengeClaim", foreign_key: 'request_prompt_id'
+  has_many :request_claims, class_name: "ChallengeClaim", foreign_key: "request_prompt_id", inverse_of: :request_prompt
```

Comment:
Should I do this? It would be `dependent: :destroy`, and it would let us delete this method:
https://github.com/otwcode/otwarchive/blob/27d089d5189069609eeea552e6a7d2ba52c7917e/app/models/prompt.rb#L49-L53
But it might require more manual QA.

Reply:
Let's do this. For manual QA, do we delete the sign-up and make sure the related claims disappear?

Reply:
Yeah, that's what I'd do for QA. (And maybe also try deleting a prompt with a fulfilled claim, just to make sure it doesn't cause issues for the work.)